aws-android-sdk-kinesisvideo/build.gradle

```diff
@@ -22,7 +22,9 @@ dependencies {
         exclude group: 'com.google.android', module: 'android'
     }
     implementation 'androidx.annotation:annotation:1.1.0'
+    implementation 'com.squareup.okhttp3:okhttp:3.9.0'
```

Comment:
You decided _not_ to add this for now, right?  If so, can you remove it from the build.gradle?

Reply:
Oops, sorry. It was accidentally included.

Reply:
Updated